config/channels/in-memory-channel/200-dispatcher-clusterrole.yaml

```diff
@@ -33,3 +33,10 @@ rules:
       - get
       - list
       - watch
+# Updates the status to reflect subscribable status.
```

Comment:
do we need something for `finalizers`, like:

https://github.com/knative/eventing/blob/01fcacd09e5a9d6ce94c4ae4e58034e2920f0bc0/config/provisioners/in-memory-channel/in-memory-channel.yaml#L48-L54

Reply:
My changes should not require anything about that, just that the dispatcher needs to update /status. Or mayhaps I misunderstand what you meant.